operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -24,9 +24,10 @@ spec:
 #  installation_tag: ""
```

Comment:
In kiali_cr.yaml and defaults/main.yaml we need to add kiali.* to the list of excluded namespaces because now the kiali-operator namespace is showing up in the namespace list.

Reply:
done